src/daily_output.ts

```diff
@@ -240,10 +247,14 @@ export class DailyOutput {
 		const queryDate = new Date(queryTimestamp);
 		const previousTimestamp = this.latestTimestamp * 1000;
 		const previousDate = new Date(previousTimestamp);
```

Comment:
`previousDate` should be the date from the last transaction that we processed and not the last time that we ran the script.